corehq/apps/api/tests/lookup_table_resources.py

```diff
@@ -240,7 +240,7 @@ def test_update(self):
 
 class TestLookupTableItemResource(APIResourceTest):
     resource = LookupTableItemResource
-    api_name = 'v0.5'
+    api_name = 'v0.6'
```

Comment:
are we to retain tests for v0.5 and add new ones for v0.6 to ensure it continues to work as expected?